x/axelarnet/module.go

```diff
@@ -395,3 +400,25 @@ func extractTokenFromAckOrTimeoutPacket(packet channeltypes.Packet) sdk.Coin {
 
 	return sdk.NewCoin(trace.IBCDenom(), amount)
 }
+
+// Temporary logic to handle in-transit IBC transfers during upgrade. Previously IBC transfers sent from asset escrow address; now sent from IBC account.
+// Deprecated: Remove this function after the v1.1 upgrade
```

Comment:
Can we remove this logic though? Based on our discussion, timeouts/ack errors could be submitted indefinitely into the future? We could remove it, but we need to query and ensure that no Axelar initiated IBC transfer is pending (there'll be probably be some dead ibc channels so it's a bit of an effort to check)

Reply:
Yes, need some effort but doable. should aim to get rid of this patch and make core logic clean